src/pycountry/db.py

```diff
@@ -74,12 +81,12 @@ def _load(self):
             self.objects.append(obj)
             # Inject into index.
             for key, value in entry.items():
-                # Lookups and searches are case insensitive. Normalize
-                # here.
-                value = value.lower()
                 if key in self.no_index:
                     continue
+                # Lookups and searches are case insensitive. Normalize
+                # here.
                 index = self.indices.setdefault(key, {})
+                value = value.lower()
```

Comment:
Only converting the `value` to lowercase if we're actually interacting with the index. Should be a (super tiny) performance improvement